icechunk/src/repository.rs

```diff
@@ -922,20 +907,18 @@ async fn get_existing_node<'a>(
     }
 }
 
-async fn distributed_flush<I: IntoIterator<Item = ChangeSet>>(
+async fn flush(
     storage: &(dyn Storage + Send + Sync),
-    change_sets: I,
+    change_set: &mut ChangeSet,
```

Comment:
I don't understand why this needs a mutable ref.

Reply:
It was a relic from the distributed flush! Good catch!